search/filters/FulltextFilter.php

```diff
@@ -78,13 +82,14 @@ public function getDbName() {
 	 * Adds table identifier to the every column.
 	 * Columns must have table identifier to prevent duplicate column name error.
 	 *
+	 * @param array $columns
 	 * @return string
-	*/
+	 */
 	protected function prepareColumns($columns) {
 		$cols = preg_split('/"?\s*,\s*"?/', trim($columns, '(") '));
-		$class = ClassInfo::table_for_object_field($this->model, current($cols));
-		$cols = array_map(function($col) use ($class) {
-			return sprintf('"%s"."%s"', $class, $col);
+		$table = DataObject::getSchema()->tableForField($this->model, current($cols));
+		$cols = array_map(function($col) use ($table) {
+			return sprintf('"%s"."%s"', $table, $col);
```

Comment:
I really should not have called this method quotedColumn. The intent of that method is to match a class to a specific table where a given field is declared. I can see the confusion this has caused.

Maybe I should just use tableForField() where applicable and make users continue to ansi-sql quote their identifiers.